src/core/simulation.jl

```diff
@@ -128,18 +138,18 @@ function _get_simulation_initial_times!(sim::Simulation)
             stage_initial_times[stage_number] = PSY.generate_initial_times(stage_system,
                                                         get_interval(get_sequence(sim), stage_name),
                                                         get_horizon(get_sequence(sim), stage_name))
-            isempty(stage_initial_times[stage_number]) ? throw(ArgumentError("Simulation interval 
-                                    and forecast interval definitions are not compatible")) : nothing ; 
+            isempty(stage_initial_times[stage_number]) ? throw(IS.ConflictingInputsError("Simulation interval ($(get_interval(get_sequence(sim), stage_name))) and 
```

Comment:
nit:  repeated calls to get_interval(get_sequence).  Also, go please ahead and remove the ternary operator.